### PR TITLE
Enhance data export

### PR DESCRIPTION
**Pull request by truongdd03**
# Description

Shows end time and fixes the error in raw export.

Partly Addresses #599 

## Type of change

- [ ] Note merging this changes the node modules
- [ ] Note merging this changes the database configuration.
- [x] This change requires a documentation update

## Checklist

- [x] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [x] I have removed text in ( ) from the issue request

## Limitations

The export file still requires to remove the first column (meter's name) to be importable where each exported meter needs to be in its own file.

### CODE REVIEW

File: src/client/app/utils/exportData.ts [hunk 1]
```diff
@@ -78,10 +64,11 @@ export default function graphExport(dataSets: ExportDataSet[], name: string) {
  */
 export function downloadRawCSV(items: RawReadings[], defaultLanguage: string) {
 	// note that utc() is not needed
-	let csvOutput = 'Label,Readings,Start Timestamp\n';
+	let csvOutput = 'Label,Readings,Start Timestamp,End Timestamp\n';
 	items.forEach(ele => {
-		const timestamp = moment(ele.startTimestamp).format('dddd LL LTS').replace(/,/g, ''); // use regex to omit pesky commas
-		csvOutput += `"${ele.label}",${ele.reading},${timestamp}\n`; // TODO: add column for units
+		const startTimestamp = ele.startTimestamp;
```
Comment (huss) [inline src/client/app/utils/exportData.ts]: I think the moment, format, replace that was removed for startTimestamp and endTimestamp should be put back as they are now for convertToCSV() so it works the same way when the language is changed.

**Review by huss — CHANGES_REQUESTED** [review]
There is the only one comment I have on this draft pull request. This works as expected and cleans up the code. I appreciate @truongdd03 for being so through. Once any concerns the author has are resolved then the draft state can be removed.

File: src/client/app/utils/exportData.ts (hunk 1 above)
Comment (truongdd03) [reply src/client/app/utils/exportData.ts]: Thank you for your comment. I have fixed it in the new commit.

**Review by huss — APPROVED** [review]
Looks great. Ready to merge.

**Pull request merged**